src/JWT/Algorithms/RS256Algorithm.cs

```diff
@@ -64,11 +64,19 @@ public byte[] Sign(byte[] key, byte[] bytesToSign)
         /// <param name="bytesToSign">The bytes to sign.</param>
         /// <returns>The signed bytes.</returns>
         public byte[] Sign(byte[] bytesToSign) =>
+#if NET35
+            ((RSACryptoServiceProvider)_privateKey).SignData(bytesToSign, "SHA256");
+#else
             _privateKey.SignData(bytesToSign, HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1);
+#endif
 
         /// <inheritdoc />
         public bool Verify(byte[] bytesToSign, byte[] signature) =>
+#if NET35
+            ((RSACryptoServiceProvider)_publicKey).VerifyData(bytesToSign, "SHA256", signature);

```

Comment:
Second parameter can be `HashAlgorithmName.SHA256.ToString()`

Reply:
Or it can be another property, like `HashingAlgorithmName` and then used in all methods wherever necessary. Probably the latter is what I'd do.

Reply:
HashAlgorithmName class does not exist in net35 world, therefore I went for the simplest thing that works in this case.

Reply:
I mean it can be just a string property, reused by the methods. Rather than reusing same string every time. Would be more readable and less error-prone, imo.